packages/odyssey-react/src/components/Status/Status.module.scss

```diff
@@ -14,15 +14,13 @@ $status-indicator-size: 1em * 0.5;
 
 .label {
   display: block;
-  margin-block-end: $spacing-xs;
```

Comment:
Removing this causes a small regression. Add it back and good to go!

Reply:
Added back in.